config/crd/bases/opentelemetry.io_opentelemetrycollectors.yaml

```diff
@@ -282,6 +282,7 @@ spec:
                 type: string
               tolerations:
                 description: Toleration to schedule OpenTelemetry Collector pods.
+                  This is only relevant to daemonsets, statefulsets and deployments
```

Comment:
@jpkrohling I just noticed in my local setup. I missed committing this file after adding the commect line in `opentelemetry_types.go` file.

Reply:
Yeah, we should have a CI check for this situation. Basically, generate the manifests and fail the build if there are changes.

Reply:
I just reverted this change, as it belongs elsewhere.